src/nupic/algorithms/KNNClassifier.py

```diff
@@ -356,7 +356,9 @@ def learn(self, inputPattern, inputCategory, partitionId=None, isSparse=0,
     @param inputCategory (int) The category to be associated to the training
         pattern
 
-    @param partitionId (int) UNKNOWN
+    @param partitionId (int) The k-fold cross validation partition for this
```

Comment:
I agree, but my point is we shouldn't assume an implementation :wink: 
So maybe we can instead say `During inference all training vectors with partitionId equal to that of the test vector are ignored, e.g. when switching between train and test partitions in k-folds cross validation.`